x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -73,31 +74,57 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
   });
 
   it('Shows endpoint on the list', () => {
-    loadPage(APP_ENDPOINTS_PATH);
-    cy.contains('Hosts running Elastic Defend').should('exist');
-    cy.getByTestSubj(AGENT_HOSTNAME_CELL)
-      .contains(createdHost.hostname)
-      .should('have.text', createdHost.hostname);
+    beforeChain.then(() => {
+      loadPage(APP_ENDPOINTS_PATH);
+      cy.contains('Hosts running Elastic Defend').should('exist');
+      cy.getByTestSubj(AGENT_HOSTNAME_CELL)
+        .contains(createdHost.hostname)
+        .should('have.text', createdHost.hostname);
+    });
   });
 
-  describe('Endpoint reassignment', () => {
-    let response: IndexedFleetEndpointPolicyResponse;
-    let initialAgentData: Agent;
+  it('should update endpoint policy on Endpoint', () => {
+    const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
 
-    before(() => {
-      getAgentByHostName(createdHost.hostname).then((agentData) => {
-        initialAgentData = agentData;
-      });
-      getEndpointIntegrationVersion().then((version) => {
-        createAgentPolicyTask(version).then((data) => {
-          response = data;
+    loadPage(APP_ENDPOINTS_PATH);
+
+    beforeChain.then(() => {
```

Comment:
Yeah, this just seems like an anti-pattern for cypress. the `beforeChain` should have already completed by the time the tests are executed.

Reply:
Ah, you're right. I misunderstood the cypress docs.